### PR TITLE
Create CP43.md

### PR DESCRIPTION
**Pull request by Rhanok09**
Added CP43

### CODE REVIEW

File: cps/CP43.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+1) So that we could support new tokens through XCM by simply registering them in the asset registry instead of hardcoding them in the CurrencyId type and thus not require runtime upgrades to support new tokens.
+2) So that we had a standardised way of querying metadata for all tokens that Centrifuge handles, i.e, both native tokens like CFG and Tranche tokens, as well as "foreign" tokens like AUSD, USDC, etc.
+ 
+This motion is now a step into cleaning up that first legacy, "hardcoding"-based approach; we do that by dropping those hardcoded currency variants (```AUSD``` and ```KSM``` (the latter only applicable on Altair)) and have them registered as ```Currency::ForeignAsset(<asset_id>)``` in the asset registry instead. The next step will be to migrate all balances that are now under ```CurrencyId::AUSD```  and ```CurrencyId::KSM``` to ```CurrencyId::ForeignAsset(<aUSD_asset_id>)``` and ```CurrencyId::ForeignAsset(<ksm_asset_id>)```, respectively.
+
+Proposal hash: ```0x6de31686f6a4fbf1584f91e45961862f44e5a692b23a122bc29e26c5e2275d60```
+
+GitHub issue covering this work: https://github.com/centrifuge/centrifuge-chain/issues/826
+
+## Governance process for this proposal
```
Comment (ImdioR) [inline cps/CP43.md:26]: Should we add also the step:
-  Create a Proposal post forum?

File: cps/CP43.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+1. Council Motion
+2. Referendum
```
Comment (ImdioR) [inline cps/CP43.md:28]: Public referendum ?

File: cps/CP43.md (hunk 1 above)
Comment (Rhanok09) [reply cps/CP43.md:26]: I don't think it's relevant to mention the Proposal Post in the Governance process for this CP - we haven't done it in the previous CPs (RUs). See [here](https://github.com/centrifuge/cps/blob/main/cps/CP26.md).

File: cps/CP43.md (hunk 2 above)
Comment (Rhanok09) [reply cps/CP43.md:28]: All referenda are public but I can add it to keep the wording consistent with the steps in the process.

**Comment by Rhanok09** [comment]
This CP will part of the next RU instead so closing this PR.

**Pull request closed without merging**